policy_humanoid:
  decimation: 10


inference:
  policy_type: "Humanoid"
  model_path: "/home/wsy/Library/ovinf/model/humanoid.onnx"
  log_data: true
  log_dir: "/home/wsy/log/policy"
  device: "CPU"
  cycle_time: 0.84
  single_obs_size: 47
  obs_buffer_size: 15
  action_size: 12
  action_scale: 0.5
  obs_scales:
    ang_vel: 1.0
    lin_vel: 2.0
    command: 1.0
    dof_pos: 1.0
    dof_vel: 0.05
    proj_gravity: 1.0
  clip_action: 18.0
  realtime: false

  # The order below matters
  policy_joint_names:
    - "l_hip_p"
    - "l_hip_r"
    - "l_hip_y"
    - "l_knee"
    - "l_ankle_p"
    - "l_ankle_r"
    - "r_hip_p"
    - "r_hip_r"
    - "r_hip_y"
    - "r_knee"
    - "r_ankle_p"
    - "r_ankle_r"

  policy_default_position:
    l_hip_p: 0.0
    l_hip_r: 0.0
    l_hip_y: 0.0
    l_knee: 0.0
    l_ankle_p: 0.0
    l_ankle_r: 0.0
    r_hip_p: 0.0
    r_hip_r: 0.0
    r_hip_y: 0.0
    r_knee: 0.0
    r_ankle_p: 0.0
    r_ankle_r: 0.0

